src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -81,6 +76,16 @@ public UniquePersonList(UniquePersonList source) {
         return Collections.unmodifiableList(internalList);
     }
 
+    /**
+     * Returns an unmodifiable java List view with elements cast as immutable {@link ReadOnlyPerson}s.
+     * For use with other methods/libraries.
+     * Any changes to the internal list/elements are not immediately visible in the returned list.
+     */
+    public List<ReadOnlyPerson> immutableSortedListView() {
```

Comment:
The comment appears to the same as immutableListView except for the last sentence. Perhaps you can mention that it returns a list view sorted according to alphabetical order?